src/main/java/rs/teslaris/core/dto/document/DocumentFileDTO.java

```diff
@@ -0,0 +1,36 @@
+package rs.teslaris.core.dto.document;
+
+import java.util.List;
+import javax.validation.Valid;
+import javax.validation.constraints.NotNull;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+import org.springframework.web.multipart.MultipartFile;
+import rs.teslaris.core.dto.commontypes.MultilingualContentDTO;
+import rs.teslaris.core.model.document.License;
+import rs.teslaris.core.model.document.ResourceType;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@ToString
+public class DocumentFileDTO {
+
+    private Integer id;
+
+    @NotNull(message = "You must provide a valid proof file.")
```

Comment:
again, I think DocumentFile is not just for proofs, it might be later for CVs, Theses, journal articles, research datasets, etc.